src/Advanced addition, mutiplication et inequality world.tex

```diff
@@ -0,0 +1,101 @@
+\documentclass[french,frenchkw]{article}
+\usepackage[utf8]{inputenc}
+\usepackage{natbib}
+\usepackage{babel}
+\usepackage{graphicx}
+\usepackage{amsmath}
+\usepackage{tabto}
+\usepackage[T1]{fontenc}
+
+\graphicspath{{./Images/}}
+
+
+\title{Projet de Mathématiques : Assistant de Preuves}
+\author{Ivan Hasenohr (pour cette partie)}
+\date{Mars 2020}
+\begin{document}
+\maketitle
+\paragraph{Exact, Intro, Have, Apply}
+Ici nous allons présenter 4 techniques fondamentales pour l'utilisation de fonctions, une fonction f : $A\to B$ pour A et B deux types étant simplement un élément de type $A\to B$, qui à une preuve de A renvoie une preuve de B.\\
+\textbf{Exact}\\
+La première de ces tactiques est \textit{exact}. Elle permet de dire à Lean que le but recherché correspond exactement à ce que vous lui indiquez. Par exemple, si le but est  $\exists  p \in P$, et que vous disposez de $p \in P$, alors \textit{exact p,} terminera la preuve. De même, si le but est  $\exists  q \in Q$ et que vous disposez d'un élément p de type P et d'une fonction f : $P \to Q$, alors \textit{exact f(p),} terminera la preuve.\\
+\textbf{Intro} \\
+Lorsque vous manipulez des fonctions, Lean peut vous demander de créer une fonction d'un type P vers un type Q. Une méthode est alors de d'émettre l'hypothèse qu'on dispose d'un $p\in P$ à partir duquel vous fabriquerez un élément de Q. \textit{intro p,} fait cela : vous disposerez alors d'une preuve p de P et votre but sera reformulé en Q. \\
```

Comment:
Fais attention, `p \in P` ça veut dire: p est dans l'ensemble|chose P tandis que `p: P` c'est `p` est de type`P`, ce n'est pas la même chose hélas

Donc, parle plutôt de `p: P` que de `p \in P` en expliquant que ça a un sens similaire mais que nous ne sommes pas en théorie des ensembles donc `p \in P` est plus proche du monde ensembliste que `p: P`, tu n'écris pas `p \in (\forall x \in X, x = 0)` par exemple. Pourtant, c'est un peu ce qu'on fait (et ce que tu écris).